### DISCUSSION TITLE
Can I improve low-light performance of the model which is trained on daytime images?

### DISCUSSION THREAD

**Discussion by vinaysanga** [ultralytics/ultralytics#19355] [Q&A]
I have a Yolo 11 trained on a custom dataset with daytime images. Now, I want to improve its low-light or night-time performance. What are the things that I can do without needing it to training from scratch?
More info:
1. I have a few low-light samples annotated now. Can I somehow add this info on top of the existing model?
2. I trained the model using only low-light samples on the existing daytime model. It completely forgot how to predict in daytime.
3. I can go for a training from scratch, but it would take a lot of time. So, modifying the existing model would be the best choice.

**Comment by UltralyticsAssistant** [comment]
👋 Hello @vinaysanga, thank you for your interest in Ultralytics 🚀! Improving model performance in low-light conditions is an intriguing challenge, and we're here to help guide you in the right direction.

We recommend going through our [Docs](https://docs.ultralytics.com/) for helpful insights on training and fine-tuning, especially the [Model Training Tips](https://docs.ultralytics.com/guides/model-training-tips/). Based on your description, there are a few strategies you can explore without starting from scratch.

If this is a 🐛 Bug Report:
Please provide additional context, such as console output, errors, or a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/). A clear MRE helps us debug the issue efficiently.

❓ For questions regarding custom fine-tuning (such as yours):
1. Using the annotated low-light images, you can consider **fine-tuning** your current model instead of training it from scratch. Fine-tuning typically involves using a smaller `learning_rate` and may require careful `Class-Weights` balancing to prevent overfitting to the low-light samples.
2. Monitor for **catastrophic forgetting** (as you’ve already observed) by experimenting with the dataset split. You can mix a subset of daytime samples with the low-light images during fine-tuning to help the model retain daytime capabilities.
3. Leverage **augmentation techniques** (e.g., brightness or gamma adjustments) to simulate low-light conditions during training. This can be easily integrated into your pipeline.

🤓 For real-time community advice, feel free to join [Discord](https://discord.com/invite/ultralytics) 🎧. Looking for detailed insights? Post on [Discourse](https://community.ultralytics.com/). Or connect with the YOLO fanbase on our [Subreddit](https://reddit.com/r/Ultralytics).

## Upgrade  
Ensure you're using the latest version of the `ultralytics` package and all dependencies to provide the best foundation:

```bash
pip install -U ultralytics
```

## Environments  
For seamless experimentation, try running your workflows in verified environments below, preconfigured with dependencies:

- **Notebooks**: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud**: Deep Learning VMs ([Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/))
- **Amazon Cloud**: DL AMIs ([Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/))
- **Docker**: ([Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/)) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status  
<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are passing. CI ensures reliable operation across YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on major platforms like macOS, Windows, and Ubuntu.

This is an automated response to get you started. An Ultralytics engineer will assist you further soon 😊.

**Reply by vinaysanga** [reply]
@UltralyticsAssistant How to integrate data augmentation techniques such as brightness or gamma adjustment now in the pipeline?
Also, what is the correct way to fine-tune the model?
